packages/tools/pie-icons-vue/rollup.config.js

```diff
@@ -0,0 +1,39 @@
+import del from 'rollup-plugin-delete';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
+export default [
```

Comment:
You could share this config and override what you need doesn't have to happen in this pr tho.

You could share this and the next one in a shared config and then pass it into each.

```javascript
const sharedConfig = {
    input: ['generated/index.js'],
    output: {
        exports: 'named',
        dir: '',
        format: '',
        name: '@justeattakeaway/pie-icons-vue',
        preserveModulesRoot: 'icons',
        entryFileNames: '[name].[format]',
        preserveModules: true,
    },
    plugins: [],
};
```
```javascript

export default [
    {
        ...sharedConfig,
        output: {
            ...sharedConfig.output,
            dir: 'esm' ...
        }
    },
    {
       ...sharedConfig,
        output: {
            ...sharedConfig.output,
            dir: 'dist' ...
        }
     }
]
```

Reply:
yeah good point actually, would be worth looking into making it shared now rollup is being used by all icon packages 👍 I'll tackle this in a separate PR. Thank you!